src/renderers/dom/fiber/ReactDOMFiber.js

```diff
@@ -352,6 +352,10 @@ var ReactDOM = {
   },
 
   unmountComponentAtNode(container : DOMContainerElement) {
+    invariant(
+      isValidContainer(container),
+      'unmountComponentAtNode(...): Target container is not a DOM element.'
+    );t
```

Comment:
Maybe a better fix would just be to update the `ReactMount-test` to expect the slightly different error message based on `ReactDOMFeatureFlags.useFiber`?